BUILD.txt

```diff
@@ -1,6 +1,6 @@
 The instructions on how to build the SSSD and contribute to the
 project can be found here:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-https://sssd.github.io/developers/index.html
+https://sssd.io/developers/index.html
```

Comment:
This link returns 404 right now.
I think the valid one may be: https://sssd.io/docs/developers/